app/views/summaries/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>New Summary</h1>
+<%= form_for [@summary], url: topic_post_summaries_path(@topic, @post, @summary) do |f| %>
+<%= f.label :description %>
```

Comment:
Please indent everything within the `form_for`.  The `do` at the end is a hint you are starting a block and the `end` represented the end of the block. Everything in between should be indented.